src/main/scala/IsRetweetClassifier.scala

```diff
@@ -0,0 +1,63 @@
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.spark.ml.Pipeline
+import org.apache.spark.ml.classification.LogisticRegression
+import org.apache.spark.ml.feature.{HashingTF, Tokenizer}
+import org.apache.spark.sql.functions._
+import TweetsNormalization.normalizeTweets
+import FeaturesExtractionFromRawTweet.extractFeaturesFromRawTweet
+
+object IsRetweetClassifier extends LazyLogging with SparkSessionWrapper with ConfigurationWrapper {
+  def main(args: Array[String]) {
+    import spark.implicits._
+
+    val text = spark.read.textFile(config.getString("input.path"))
+    val aggregatedText = text.agg(concat_ws("\n", collect_list("value"))).as[String]
+
+    val tweets = aggregatedText
+      .flatMap(line => line.split("List\\("))
+
+    val nonEmptyTweets = tweets
```

Comment:
all steps between  `text` and `structuredData ` are parts of a single ETL process, dedicated to data cleansing / preparation. So, the whole process should be tested.

i.e. I'd like to see a test that checks if `structuredData` was correctly created from  `text` / raw input, taking into account how raw data schema parsing and "empty" tweets are handled in this case.

NOTE: such test may require encapsulation of multiple ETL steps into a single function, so it can be tested as a unit.